<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="m_pModeWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Calendar">
       <attribute name="title">
        <string>Kalendarz</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableView" name="m_pCalendar"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="m_pTrainingDetailsLabel">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Charts">
       <attribute name="title">
        <string>Wykresy</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Wykresy</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QCheckBox" name="m_pTSSChart">
               <property name="text">
                <string>TSS</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="m_pDuration">
               <property name="text">
                <string>Czas trwania</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="m_pRestChart">
               <property name="text">
                <string>Odpoczynek</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="m_pCaloriesChart">
               <property name="text">
                <string>Kalorie</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="m_pHrRestChart">
               <property name="text">
                <string>Tętno spoczynkowe</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="m_pAvgPulseChart">
               <property name="text">
                <string>Średnie tętno</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="m_pAvgPaceChart">
               <property name="text">
                <string>Średnie tempo</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_pClearChartTips">
               <property name="text">
                <string>Wyczyść dymki</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QtCharts::QChartView" name="m_pChartView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Statistics">
       <attribute name="title">
        <string>Statystyki</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTreningi">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="m_pActionQuit"/>
   </widget>
   <addaction name="menuTreningi"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="m_pActionQuit">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
